hyperjumptech/monika
# Feature(config): [295] return after create configuration file

# Monika Pull Request (PR)  
fix #295 
## What feature/issue does this PR add  
Monika is running the probes even though it should only create configuration file.

## How did you implement / how did you fix it  
fix logic for returning after create configuration file

## How to test  
1. run cmd `./bin/run --create-config --har copy_example.har`
2. monika should immediately return, and closed 
3. make sure monika is not running and monika.json file is created
